phpstan-baseline.neon

```diff
@@ -1,6 +1,27 @@
 parameters:
-    ignoreErrors:
-        -
-            message: "#^Class Doctrine\\\\ODM\\\\MongoDB\\\\DocumentManager not found\\.$#"
-            count: 1
-            path: src/Document/BaseDocumentManager.php
+	ignoreErrors:
+		-
+			message: "#^Strict comparison using \\=\\=\\= between null and object will always evaluate to false\\.$#"
+			count: 1
+			path: src/Adapter/ORM/DoctrineORMAdapter.php
+
+		-
+			message: "#^Strict comparison using \\=\\=\\= between null and object will always evaluate to false\\.$#"
+			count: 1
+			path: src/Adapter/PHPCR/DoctrinePHPCRAdapter.php
+
+		-
+			message: "#^Call to an undefined method Doctrine\\\\Persistence\\\\ObjectManager\\:\\:getConnection\\(\\)\\.$#"
+			count: 1
+			path: src/Document/BaseDocumentManager.php
+
+		-
+			message: "#^Access to an undefined property Doctrine\\\\Persistence\\\\Mapping\\\\ClassMetadata\\:\\:\\$table\\.$#"
+			count: 1
+			path: src/Model/BaseManager.php
```

Comment:
We probably should remove `ManagerInterface::getTableName` and `ManagerInterface::getConnection` from the interface as they seem ORM related.

https://github.com/sonata-project/sonata-doctrine-extensions/blob/5db3c1a2e6ded14cf22e97c4b521b62713f44280/src/Model/ManagerInterface.php#L88-L100